config.template.toml

```diff
@@ -165,11 +165,119 @@ model = "gpt-4o"
 #disable_vision = true
 
 [llm.gpt4o-mini]
+# Model to use
+model = "gpt-4o-mini"
 # API key to use
 api_key = "your-api-key"
 
-# Model to use
-model = "gpt-4o-mini"
+
+################################### Router ###################################
+# litellm Router configuration
+# https://litellm.vercel.app/docs/routing
+##############################################################################
+
+# The "router_config"section can only occur once in the config.toml file!
+#[router_config]
+
+# The default model to use for routing, which must be part of the "models" entries below.
+#default_model = "gpt-4o"
+
+# The strategy to use for routing (e.g., usage-based-routing, cost-based-routing)
+#routing_strategy = "usage-based-routing"
+
+# Number of retries to attempt when an operation fails
+#num_retries = 3
+
+# waits min. 5s before retrying request
+#retry_after = 15
+
+# cooldown model if it fails > 1 call in a minute
+#allowed_fails = 1
+
+# cooldown the deployment for 100 seconds if it num_fails > allowed_fails
+#cooldown_time = 100
+
+# raise timeout error if call takes > 240s
+#timeout = 240
+
+# Whether to cache responses for reuse
+#cache_responses = false
+
+# Additional cache configuration options, see documentation for details
+#[router_config.cache_kwargs]
+
+#[router_config.retry_policy]
+#exceptions_to_retry = [ "RateLimitError",]  # List of exceptions to retry on
+#max_retries = 8  # Maximum number of retries for the specified exceptions
+#retry_after = 15  # Time to wait before the first retry for the specified exceptions
```

Comment:
This is a `retry_after`... but there is another `retry_after` at line 192. I'm not sure what the difference between them is. Does the one at line 192 retry for _more_ exceptions than those in `retry_policy`?

Reply:
I started to understand this... Well, in part. Sorry, this in the doc you linked, I just jumped to the code first. 😅

Reply:
I cannot find this retry_policy in liteLLM. Their object here is: ([source](https://github.com/BerriAI/litellm/blob/6005450c8fe6f354c6d3dbcdc873ddc5c4025c70/litellm/types/router.py#L444))
```
class RetryPolicy(BaseModel):
    """
    Use this to set a custom number of retries per exception type
    If RateLimitErrorRetries = 3, then 3 retries will be made for RateLimitError
    Mapping of Exception type to number of retries
    https://docs.litellm.ai/docs/exception_mapping
    """

    BadRequestErrorRetries: Optional[int] = None
    AuthenticationErrorRetries: Optional[int] = None
    TimeoutErrorRetries: Optional[int] = None
    RateLimitErrorRetries: Optional[int] = None
    ContentPolicyViolationErrorRetries: Optional[int] = None
    InternalServerErrorRetries: Optional[int] = None
```